blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
```

Comment:
@tfennelly as said in the earlier comment the "dummy" is a fake version of that not existing plugin